ivadomed/loader/utils.py

```diff
@@ -810,14 +811,15 @@ class BidsDataframe:
 
     def __init__(self, loader_params, derivatives, path_output):
 
-        # path_data from loader parameters
+        # paths_data from loader parameters
+        # TODO: when integrating in pipeline, remove format_path_data here (done before in main)
         self.paths_data = imed_utils.format_path_data(loader_params['path_data'])
 
         # bids_config from loader parameters
         self.bids_config = None if 'bids_config' not in loader_params else loader_params['bids_config']
 
         # target_suffix and roi_suffix from loader parameters
-        self.target_suffix = loader_params['target_suffix']
+        self.target_suffix = copy.deepcopy(loader_params['target_suffix'])
```

Comment:
If I don't get it wrong, we had issues with this in the past, and then @ahill187 suggested a better way to do this, didn't you?

Reply:
@ahill187 here I use "deepcopy" because I want to append roi_suffix to target_suffix inside the function but not outside. Is it OK? Is there a better way to do this? Thanks!

Reply:
Hmmm I can't remember! `deepcopy` looks okay to me!